lib/crypto/base36.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const crypto = require('crypto')
+
+function base36(len) {
+  let out = new Array(len)
+  while (out.length < len) {
```

Comment:
This comparison is broken. You create an array of length `len`, the compare `out.length < len` - out is always empty!